tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2081 @@
+                    if iteration % 10 == 0:
+                        log.info(
+                            f"Iteration {iteration}: File checksum - {current_checksum}"
+                        )
+
+                    # Small delay between iterations
+                    time.sleep(2)
+
+                    if iteration % 20 == 0:
+                        log.info(f"Completed {iteration} I/O iterations successfully")
+
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)